core/relay/config/Fabric_Relay2.toml

```diff
@@ -32,6 +32,11 @@ hostname="localhost"
 port="9080"
 tls=false
 tlsca_cert_path="credentials/fabric_ca_cert.pem"
+[relays.Fabric_Relay2]
```

Comment:
Can you undo these changes, it may confuse people trying to read these samples config files as reference. Even I'd prefer this self reference to be removed from Fabric_Relay.toml also.